### PR TITLE
It fixes #150

### LINKED ISSUE #150: Remove ESRF references from LDAP authentication plugin
There are some ESRF references in the LDAP authentication plugin that should be removed:
https://github.com/ispyb/py-ispyb/blob/a4d0711771811326cedc7193eb4fcb63555838d7/pyispyb/app/extensions/auth/LdapAuthentication.py#L94

### PR DESCRIPTION
**Pull request by antolinos**
- Minor string formatting conversions
- Remove ESRF reference
- Update documentation
- Adding root into the LogConfig

### CODE REVIEW

File: pyispyb/config.py [hunk 1]
```diff
@@ -86,9 +86,9 @@ def get_settings() -> Settings:
 class LogConfig(BaseModel):
     """Logging configuration to be set for the server"""
 
-    LOGGER_NAME: str = "mycoolapp"
+    LOGGER_NAME: str = "py-ispyb"
     LOG_FORMAT: str = "%(levelprefix)s | %(asctime)s | %(message)s"
-    LOG_LEVEL: str = "INFO"
+    LOG_LEVEL: str = "DEBUG"
```
Comment (mgaonach) [inline pyispyb/config.py:91]: What do you think about `INFO` by default and allowing to override it via env variable? I think `DEBUG` by default seems heavy.

Comment (antolinos) [reply pyispyb/config.py:91]: Personally, I'd set the db logger to `INFO` and leave the root level to `DEBUG` (ideally with the option to be override by a configuration parameter or env)
I like seeing the logs during the development but I guess it is a question of preferences of each developer.

Comment (stufisher) [reply pyispyb/config.py:91]: Personally i'd set both to info, you want the default to be correct for production. Debug is too verbose for production in my opinion and you risk things like log files becoming too big and taking down your services.

We could add a log level to the base config, then you can easily enabled debug by setting it in your local `.env` file?

**Comment by stufisher** [comment]
I think you are missing black formatting, it might not be installed in your env as vscode is set to automatically apply it on save?

**Pull request closed without merging**